lib/sql/columns.go

```diff
@@ -15,19 +16,26 @@ func QuoteColumns(cols []columns.Column, dialect Dialect) []string {
 	return result
 }
 
+func QuoteTableAliasColumn(tableAlias constants.TableAlias, column columns.Column, dialect Dialect) string {
```

Comment:
Does it need to be public?

Reply:
Yeah I plan on using for building merge queries in the dialects.